daemons/gptp/common/avbts_port.hpp

```diff
@@ -234,6 +234,9 @@ class IEEE1588Port {
 	static const int64_t INVALID_LINKDELAY = 3600000000000;
 	static const int64_t NEIGHBOR_PROP_DELAY_THRESH = 800;
 	static const unsigned int SYNC_RECEIPT_THRESH = 5;
+	static const unsigned int SEQID_ASCAPABLE_THRESHOLD = 5;
+	static const uint16_t LOSTPDELAY_RESP_THRESH = 3;
```

Comment:
Why aren't these called DEFAULT_SEQID_ASCAPABLE_THRESHOLD etc.?
They are really just initial values, correct?
They are overridden by the .ini file - at least that is what I assume.